pyiron_atomistics/gpaw/pyiron_ase.py

```diff
@@ -103,7 +107,7 @@ def __init__(self, project, job_name):
 
     @property
     def structure(self):
-        return GenericInteractive.structure.fget(self)
+        return ase_to_pyiron(GenericInteractive.structure.fget(self))
```

Comment:
While converting to an pyiron structure fixes the Murnaghan bug it breaks the gpaw calculation.